### PR TITLE
[SofaMatrix] More details in the readme file

### PR DESCRIPTION
**Pull request by alxbilger**
______________________________________________________

By submitting this pull request, I acknowledge that  
**I have read, understand, and agree [SOFA Developer Certificate of Origin (DCO)](https://github.com/sofa-framework/sofa/blob/master/CONTRIBUTING.md#sofa-developer-certificate-of-origin-dco)**.
______________________________________________________

**Reviewers will merge this pull-request only if**  
- it builds with SUCCESS for all platforms on the CI.
- it does not generate new warnings.
- it does not generate new unit test failures.
- it does not generate new scene test failures.
- it does not break API compatibility.
- it is more than 1 week old (or has fast-merge label).

### CODE REVIEW

**Comment by alxbilger** [comment]
See the result [here](https://github.com/sofa-framework/sofa/blob/ab67a65a56a7c4d6ec04263de9b829e30b4f5ce1/applications/plugins/SofaMatrix/SofaMatrix.md)

File: applications/plugins/SofaMatrix/SofaMatrix.md [hunk 1]
```diff
@@ -1,3 +1,60 @@
...
+
+## Components
+
+The plugin contains the following components that can be placed into a SOFA scene:
+
+### GlobalSystemMatrixExporter
+
+GlobalSystemMatrixExporter is a component to export the matrix of a linear system.
+It supports the following file formats:
+* **txt**: exports a Python-like list of list, containing all the matrix values including the zeros.
```
Comment (epernod) [inline applications/plugins/SofaMatrix/SofaMatrix.md]:
```suggestion
* **txt**: exports a Python-like list of lists, containing all the matrix values including the zeros.
```

File: applications/plugins/SofaMatrix/SofaMatrix.md [hunk 2]
```diff
@@ -1,3 +1,60 @@
...
+  pixel means the value is non-zero.
+
+GlobalSystemMatrixExporter must have a link to a linear solver (a component inhereting from `sofa::core::behavior::LinearSolver`).
+
+The link can be explicit. Example:
+```xml
+<GlobalSystemMatrixExporter linearSolver="@pathToLinearSolver"/>
+```
+
+The link can be implicit. Example:
```
Comment (epernod) [inline applications/plugins/SofaMatrix/SofaMatrix.md]:
```suggestion
The link can be implicit. I.e the component will search for a linear solver in the current Node. Example:
```

File: applications/plugins/SofaMatrix/SofaMatrix.md [hunk 3]
```diff
@@ -1,3 +1,60 @@
...
+Qt widget.
+
+GlobalSystemMatrixImage must have a link to a linear solver (a component inhereting from `sofa::core::behavior::LinearSolver`).
+
+The link can be explicit. Example:
+```xml
+<GlobalSystemMatrixImage linearSolver="@pathToLinearSolver"/>
+```
+
+The link can be implicit. Example:
```
Comment (epernod) [inline applications/plugins/SofaMatrix/SofaMatrix.md]:
```suggestion
The link can be implicit. I.e the component will search for a linear solver in the current Node. Example:
```

File: applications/plugins/SofaMatrix/SofaMatrix.md [hunk 4]
```diff
@@ -1,3 +1,60 @@
...
\ No newline at end of file
+SofaMatrix is a SOFA plugin containing components related to linear system matrices.
+
+## Components
+
+The plugin contains the following components that can be placed into a SOFA scene:
+
+### GlobalSystemMatrixExporter
+
+GlobalSystemMatrixExporter is a component to export the matrix of a linear system.
```
Comment (epernod) [inline applications/plugins/SofaMatrix/SofaMatrix.md]: Could you add where is the matrix exported by default?  and if it is possible to change the export path?

**Review by epernod — CHANGES_REQUESTED** [review]

File: applications/plugins/SofaMatrix/SofaMatrix.md (hunk 4 above)
Comment (alxbilger) [reply applications/plugins/SofaMatrix/SofaMatrix.md]: Very good remark. Thanks

**Review by epernod — APPROVED** [review]

**Pull request merged**